sdk/nodejs/iam/documents.ts

```diff
@@ -96,27 +119,33 @@ export interface PolicyStatement {
  * lets you write a condition using the s3:VersionId key, which is unique to that service.
  */
 export interface Conditions {
-    // A map of condition operators and their arguments. Condition operators are the "verbs" of conditions and specify
-    // the type of comparison that IAM performs. The condition operators are grouped into the following categorties:
-    //
-    //     - String
-    //     - Numeric
-    //     - Date and time
-    //     - Boolean
-    //     - Binary
-    //     - IP address
-    //     - Amazon Resource (ARN) (available for some services)
-    //     - ...IfExists (checks if the key value exists as part of another check)
-    //     - Null check (checks if the key value exists as a standalone check)
-    //
-    // For details on all of the available operators and their arguments, please refer to the AWS documentation:
-    // https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html.
+    /**
+     *
+     * A map of condition operators and their arguments. Condition operators are the "verbs" of conditions and specify
+     * the type of comparison that IAM performs. The condition operators are grouped into the following categorties:
+     *
+     *     - String
+     *     - Numeric
+     *     - Date and time
+     *     - Boolean
+     *     - Binary
+     *     - IP address
+     *     - Amazon Resource (ARN) (available for some services)
+     *     - ...IfExists (checks if the key value exists as part of another check)
+     *     - Null check (checks if the key value exists as a standalone check)
+     *
+     * For details on all of the available operators and their arguments, please refer to the AWS documentation:
+     * https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html.
+     */
     [operator: string]: ConditionArguments;
 }
 
 export interface ConditionArguments {
-    // e.g. { "aws:MultiFactorAuthPresent": "true" }
-    //   or { "aws:SourceIp": [ "203.0.113.0/24", "2001:DB8:1234:5678::/64" ]}
+    /**
+     *
+     * e.g. { "aws:MultiFactorAuthPresent": "true" }
+     *   or { "aws:SourceIp": [ "203.0.113.0/24", "2001:DB8:1234:5678::/64" ]}
```

Comment:
How about something like:

```suggestion
     * Examples:
     *  - `{ "aws:MultiFactorAuthPresent": "true" }`
     *  - `{ "aws:SourceIp": [ "203.0.113.0/24", "2001:DB8:1234:5678::/64" ]}`
```